server/php-queries.cpp

```diff
@@ -1089,23 +1086,20 @@ void php_queries_finish() {
 
 const char *net_event_t::get_description() const noexcept {
   static char BUF[10000];
-  switch (type) {
-    case net_event_type_t::rpc_answer: {
-      sprintf(BUF, "RPC RESPONSE: TL magic = 0x%08x, bytes length = %d", result_len >= 4 ? *reinterpret_cast<unsigned int *>(result) : 0, result_len);
-      break;
-    }
-    case net_event_type_t::rpc_error: {
-      sprintf(BUF, "RPC ERROR: error code = %d, error message = %s", error_code, error_message);
-      break;
-    }
-    case net_event_type_t::job_worker_answer: {
-      if (job_result) {
-        sprintf(BUF, "JOB RESPONSE: class name = %s", job_result->response.get_class());
+  std::visit(overloaded{
+    [](const net_events_data::rpc_answer &e) {
```

Comment:
clang-tidy complains like:
"Parameter name 'e' is too short, expected at least 3 characters"
can we fix that?

Reply:
fixed